index.html

```diff
@@ -2,10 +2,23 @@
 <html lang="en">
 
 <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
```

Comment:
- [ ] I recommend you adding `position: fixed ` property to the mobile navbar, otherwise users can scroll the screen without closing the navbar. 
- [ ] I recommend you giving `overflow: hidden` property to the body when `navbar` is open, this will also prevent users from scrolling the body section while `navbar` is open. 
- [ ] There is some white space on the top of the navbar, please make sure that the navbar takes the full screen as provided in Figma: 
<img width="359" alt="Screenshot 2022-03-07 at 15 49 53" src="https://user-images.githubusercontent.com/77038610/157029040-ec842176-cb6b-4f02-8faa-9e15b25bcaf4.png">

- [ ] Please make sure that the `navbar` takes full width and full height in mobile screens smaller than 425 px, currently more white space appears on all sides: 
<img width="321" alt="Screenshot 2022-03-07 at 15 49 03" src="https://user-images.githubusercontent.com/77038610/157028934-51e01bb7-af80-430c-9812-8d1f55cabd41.png">

Reply:
issue resolved 👍